scene/3d/skeleton.cpp

```diff
@@ -220,82 +220,89 @@ void Skeleton::_update_process_order() {
 	process_order_dirty = false;
 }
 
-void Skeleton::_notification(int p_what) {
+Transform Skeleton::compute_global_pose(const Bone &p_bone, const Bone *p_bonesptr, bool p_ignore_override) const {
+	if (!p_ignore_override && p_bone.global_pose_override_amount >= (1.0 - CMP_EPSILON)) {
+		return p_bone.global_pose_override;
+	} else {
```

Comment:
The `else` is not necessary as the previous branch returns, so you can save one level of indentation for the whole block.